swan-lake/resources/featured-scenarios/write-a-restful-api-with-ballerina.md

```diff
@@ -301,69 +325,56 @@ public type ErrorMsg record {|
 
```

Comment:
The command is not there for OpenAPI generation. Also should we add this as bullet points to the topic. Better to have them as Tip or Note

Reply:
Updated as follows:

>**Info:** It is always a good practice to document your interfaces. However, this example has omitted documentation for brevity. Nevertheless, any production-ready API interface must include API documentation. You can also try [generating an OpenAPI specification for the written service](/learn/openapi-tool/) by executing the `bal openapi -i main.bal` command, which creates a `yaml` file in the current folder.